### DISCUSSION TITLE
EquiCode - Official Website

### DISCUSSION THREAD

**Discussion by KhushiSharma22** [EquiCode/Official-Website#1] [Ideas]
Please feel free to brainstorm new features to be added and other changes here. 
You can also raise issues for some existing features.

**Comment by saumya66** [comment]
Contact Us Page would look good if placed as the last section instead of being in the middle.  I can make a PR if folks like this !

**Comment by sanyogchangmai** [comment]
The performance of the site can be improved a lot on mobile devices by optimizing the images used in the site.
Without optimizing the images the following report was produced for a mobile device.
![prev](https://user-images.githubusercontent.com/76730995/121120635-94ccaf80-c83b-11eb-8d30-1199290ec0d3.JPG)
As it can be seen that by optimizing the images we can save an estimate of 18sec. On weaker mobile devices the rendering of images might get delayed by this much.
This problem can be solved by optimizing the images by converting the images to a more compressible image format such as **webp**.
This is the report produced when I replaced all the images with **webp** format.
![after](https://user-images.githubusercontent.com/76730995/121121003-63a0af00-c83c-11eb-998a-bedd9038a635.JPG)
As it can be seen that estimated saving comes down to around 4 sec. Thats an improvement of around 14 sec.
I can open an issue for optimizing the images. Do let me know what you all think. Feel free to suggest your views.

**Reply by KhushiSharma22** [reply]
I like how you took this further with the detailed analysis, looks good to me :)

**Reply by sanyogchangmai** [reply]
Thank you, I have opened an issue for this.
